.github/workflows/latest-kernel.yml

```diff
@@ -20,24 +20,34 @@ jobs:
       - name: Checkout Archlinux packages ⤵️
         uses: actions/checkout@v3
         with:
-          repository: 'archlinux/svntogit-packages'
-          ref: 'packages/linux'
+          repository: 'https://aur.archlinux.org/linux-mainline.git'
           path: 'linux'
     
       - name: Fetch latest kernel available on archlinux
         id: latest-version
+        # Note: in case we are building latest mainline,
+        # we grep the linux-mainline aur PKGBUILD "_tag" line, that is made like: "_tag=v6.4-rc1"
+        # We then need to extract the part after the "=" and finally remove the starting "v".
         run: |
           if [ "${{ inputs.linux-version }}" = "" ]; then
-            cd linux/trunk/
-            line=$(grep "pkgver" PKGBUILD | head -n1)
-            krel=${line#*=}
-            line=$(grep "pkgrel" PKGBUILD | head -n1)
-            kver=${line#*=}
-            echo "latest_vers=${krel}-${kver}" >> $GITHUB_OUTPUT
+            cd linux/
+            krel=$(grep "_tag" PKGBUILD | head -n1 | awk -F"=" '{print $2}')
+            echo "latest_vers=${krel:1}" >> $GITHUB_OUTPUT
+            echo "use_vanilla=true" >> $GITHUB_OUTPUT
+            echo "kernelconfig=$(cat config | base64 | tr -d '\n')" >> $GITHUB_OUTPUT
```

Comment:
I am not sure how `GITHUB_OUTPUT` will behave pushing lots of megabytes string into it. :pray: